src/test/java/org/tikv/raw/RawKVClientTest.java

```diff
@@ -102,17 +103,12 @@ public void atomicAPITest() {
     ByteString value = ByteString.copyFromUtf8("value");
     ByteString value2 = ByteString.copyFromUtf8("value2");
     client.delete(key);
-    ByteString res1 = client.putIfAbsent(key, value, ttl);
-    assert res1.isEmpty();
-    ByteString res2 = client.putIfAbsent(key, value2, ttl);
-    assert res2.equals(value);
+    client.compareAndSet(key, null, value, ttl);
     try {
-      Thread.sleep(ttl * 1000);
-    } catch (InterruptedException e) {
-      Thread.currentThread().interrupt();
+      client.compareAndSet(key, null, value2, ttl);
+    } catch (RawCASConflictException err) {
+      assert err.getCurrValue() == Optional.of(value);
     }
-    ByteString res3 = client.putIfAbsent(key, value, ttl);
-    assert res3.isEmpty();
```

Comment:
can we keep the `ttl` test here?

Reply:
First, the test is ignored. Second, this would never pass.

Reply:
@birdstorm PTAL

Reply:
In design, this test should not be ignored if we could determine that the tikv version is 5.0 and above. I suggest to leave it as it was.